backend/internal/gateway/proxy.go

```
@@ -0,0 +1,179 @@
+	if registerActive {
+		g.active.register(entry)
+		defer g.active.remove(requestID)
+	}
+
+	workerStarted := time.Now()
+	r.Body = io.NopCloser(bytes.NewReader(body))
+	r.ContentLength = int64(len(body))
+	r.Header.Del("Authorization")
+
+	proxy := httputil.NewSingleHostReverseProxy(target)
+	original := proxy.Director
```

golangci-lint (2.13.2)

[error] SA1019: (net/http/httputil.ReverseProxy).Director has been deprecated since Go 1.26 and an alternative has been available since Go 1.20: Use Rewrite instead. 

(staticcheck)